src/controleur/AjaxRequestController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+class AjaxRequestController extends Controller
+{
+
+    // Classes PDO utilisées
+    private VilleBD $villeBD;
+
+    // Variables utilisées par la vue
+
+    // Variables utilisées par le controller
+    private string  $request;
+
+    public function __construct()
+    {
+        $this->titre = '';
+        $this->view = 'component/ajaxCompletionList';
+
+        $this->request = $_POST['request'];
+
+        switch ($this->request) {
+
+            case AjaxHandler::AUTOCOMPLETE_VILLE:
+                $this->villeBD = new VilleBD();
+                $colName = 'nom';
+
+                $this->data = $this->autocomplete($this->villeBD, $colName);
+                $this->render($this->view, $this->data, false);
```

Comment:
$this->render($this->titre, $this->view, $this->data); (pour pas causer d'erreur)